ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -299,9 +359,25 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar })
             setProfileForModal(undefined);
           }}
         />
+
       </div>
 
       <PromptComponent ref={modalRef} />
+      <GenericModal
+        open={open}
+        handleClose={handleClose}
+        Content={
+          <Box sx = {style}>
+            <Typography id="transition-modal-title" variant="h6" component="h2" className={classes.modal}>
+              <span className = {classes.span}>&#9432;</span>Session Expired
+            </Typography>
+            { counter && <Typography id="transition-modal-description" className = {classes.modalText} >
+               Your session has expired.
+               You will now be redirected to login... {counter}
+            </Typography>
+            }
+          </Box>
+        } />
```

Comment:
Please extract this dialog and it's style in a separate component. They have been used twice on different pages.

Reply:
done